src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -9,180 +9,387 @@
 -->
 
 <template>
-  <div class="quick-navigation-modal">
-    <div class="container">
+  <div
```

Comment:
You may have already discussed this with Marina, but is there a reason why this modal seems to have been written mostly from scratch and doesn't utilize the existing `GenericModal` component? (There's likely a good reason, just curious)

Reply:
I'm using `GenericModal` as a wrapper for `QuickNavigationModal` and using most of its logic, I made some refactor here [3aaf55f](https://github.com/apple/swift-docc-render/pull/396/commits/3aaf55f1359ecdcd482ad389caa06ae36ed19c88) to enclose everything in one component and make it fit my use case better, there's anything specific that is not being reused properly? 🤔

Reply:
Got it. I think I just missed that. Apologies.